src/components/nav/TheHeader.vue

```diff
@@ -1,34 +1,94 @@
 <script setup lang="ts">
-  import { type RouteLocationNormalizedLoaded, RouterLink, useRoute } from 'vue-router'
+  import { type RouteLocationNormalizedLoaded, useRoute } from 'vue-router'
   import { useAuthStore, type AuthStore } from '@/stores/AuthStore'
+  import SchulPortalLogo from '@/assets/logos/Schulportal_SH_Bildmarke_RGB_Anwendung_HG_Blau.svg'
 
   const route: RouteLocationNormalizedLoaded = useRoute()
   const auth: AuthStore = useAuthStore()
 </script>
 
 <template>
-  <v-app-bar :elevation="2">
-    <v-spacer></v-spacer>
-    <RouterLink to="/">{{ $t('nav.landing') }}</RouterLink>
+  <v-app-bar
+    :elevation="2"
+    color="#001E49"
+    height="40"
+  >
+    <!-- Logo and title -->
+    <v-toolbar-title>
+      <v-row no-gutters>
+        <v-col
+          cols="auto"
+          class="mr-2"
+        >
+          <v-img
+            alt="SchulPortalLogo"
+            :src="SchulPortalLogo"
+            :width="30"
+            :height="30"
+          />
+        </v-col>
+
+        <!-- Hide this column on small screens and below -->
+        <v-col
+          cols="auto"
+          class="hidden-sm-and-down"
+        >
+          <div class="nav-title">SCHULPORTAL <span class="normal-weight">SH</span></div>
+        </v-col>
+      </v-row>
+    </v-toolbar-title>
     <v-spacer></v-spacer>
-    <RouterLink
-      to="/home"
-      v-if="auth.isAuthed"
-      >{{ $t('nav.home') }}</RouterLink
+
+    <!-- The link should be centralized somewhere else?-->
+    <v-btn :href="'https://medienberatung.iqsh.de/schulportal-sh.html'">
```

Comment:
```suggestion
    <v-btn :href="'https://medienberatung.iqsh.de/schulportal-sh.html'" data-testid="help-link">
```
Wenn ich die Entwürfe richtig verstehe, sollte dies kein Button, sondern ein Link sein. Beim Hovern erscheint die typische Farbänderung für den Button. (Gleiches gilt für den An-/abmelden Button)